README.md

```diff
@@ -394,6 +394,16 @@ The no-proxy list will automatically include entries for localhost.
 
 If you're running a Kubernetes variant, the no-proxy list will automatically include the Kubernetes API server endpoint and other commonly used Kubernetes DNS suffixes to facilitate intra-cluster networking.
 
+#### Metrics settings
+
+By default, Bottlerocket sends anonymous metrics when it boots, and once every six hours.
+This can be disabled by setting `send-metrics` to false.
+Here are the metrics settings:
+
+* `metrics.metrics-url`: the endpoint to which metrics will be sent.
```

Comment:
super-duper-uber-minor nit: Can we capitalize the first word of the descriptions just to be consistent with the rest?